File: duplicate-assert/python-unittest/JavaParserTest_testAppendHiddenBlockCommentNodes.py
import unittest

class JavaParserTest(unittest.TestCase):

    def test_append_hidden_block_comment_nodes(self):
        root = JavaParser.parse_file(File(get_path("InputJavaParserHiddenComments.java")),
                                     JavaParser.Options.WITH_COMMENTS)

        block_comment = TestUtil.find_token_in_ast_by_predicate(
            root, lambda ast: ast.get_type() == TokenTypes.BLOCK_COMMENT_BEGIN
        )

        self.assertTrue(block_comment.is_present(), "Block comment should be present")

        comment = block_comment.get()

        self.assertEqual(3, comment.get_line_no(), "Unexpected line number")
        self.assertEqual(0, comment.get_column_no(), "Unexpected column number")
        self.assertEqual("/*", comment.get_text(), "Unexpected comment content")

        comment_content = comment.get_first_child()
        comment_end = comment.get_last_child()

        self.assertEqual(3, comment_content.get_line_no(), "Unexpected line number")
        self.assertEqual(2, comment_content.get_column_no(), "Unexpected column number")
        self.assertEqual(9, comment_end.get_line_no(), "Unexpected line number")
        self.assertEqual(1, comment_end.get_column_no(), "Unexpected column number")

File: duplicate-assert/CSharpXUnit/CSharpXUnit.Tests/Tests/JavaParserTest_testAppendHiddenBlockCommentNodes.cs
using Xunit;

public class JavaParserTest : AbstractModuleTestSupport
{
    [Fact]
    public void TestAppendHiddenBlockCommentNodes()
    {
        var root = JavaParser.ParseFile(new File(GetPath("InputJavaParserHiddenComments.java")), JavaParser.Options.WithComments);

        var blockComment = TestUtil.FindTokenInAstByPredicate(root, ast => ast.Type == TokenTypes.BlockCommentBegin);

        Assert.True(blockComment.HasValue, "Block comment should be present");

        var comment = blockComment.Value;

        Assert.Equal(3, comment.LineNo, "Unexpected line number");
        Assert.Equal(0, comment.ColumnNo, "Unexpected column number");
        Assert.Equal("/*", comment.Text, "Unexpected comment content");

        var commentContent = comment.FirstChild;
        var commentEnd = comment.LastChild;

        Assert.Equal(3, commentContent.LineNo, "Unexpected line number");
        Assert.Equal(2, commentContent.ColumnNo, "Unexpected column number");
        Assert.Equal(9, commentEnd.LineNo, "Unexpected line number");
        Assert.Equal(1, commentEnd.ColumnNo, "Unexpected column number");
    }
}
